src/slick.core.ts

```diff
@@ -910,6 +918,7 @@ const SlickCore = {
     hide: hide,
     slideUp: slideUp,
     slideDown: slideDown,
+    applyDefaults: applyDefaults,
```

Comment:
I forgot to review these exports, in ES6 you can omit the right side if it's the same name as on the left. We should probably simplify them all
```diff
- applyDefaults: applyDefaults,
+ applyDefaults, 
```